src/styles/brackets_fonts.less

```diff
@@ -27,57 +27,53 @@
 @font-weight-light: 200;
 
 
-/* SourceSansPro Regular */
+/* @sansFontFamily Regular */
```

Comment:
I don't think we should change any of these `font-face` declarations, since they won't get picked up at all, based on your changes below.  Revert all this, please.